packages/connect-iframe/webpack/prod.webpack.config.ts

```diff
@@ -41,6 +41,15 @@ const config: webpack.Configuration = {
                     filename: './workers/shared-connection-worker.[contenthash].js',
                 },
             },
+            {
+                test: /sharedLoggerWorker/i,
+                loader: 'worker-loader',
+                // issuer: /workers\/workers-*/i, // replace import ONLY in /workers\/workers- not @trezor/transport
+                options: {
+                    worker: 'SharedWorker',
+                    filename: './workers/shared-logger-worker.[contenthash].js',
```

Comment:
Maybe if you remove `[contenthash]` part it will generate static file and it will allow you to reference it in popup/log.html. 
To my understanding, dynamic names are used to bypass caches but since this is not some core functionality, we should be fine without this.

Reply:
Yes, that worked, now it is loaded from same path. :+1: